src/manual/devel/best-practices/microservices.md

```diff
@@ -76,15 +81,20 @@ In the real world, that probably means a lot less.
 ### Test Setup
 
 Use Mocha to run unit tests, in the `test/` directory.
+In order to get useful stacktraces from unit tests, you should `npm install --save source-map-support`.
 Include the following in `mocha.opts`, most critically the `--ui tdd`.
```

Comment:
Maybe "critical" is extreme, but I ran into some cases where the setup/teardown functions I was using weren't being called because mocha was in the wrong ui mode.